File: GameServer/src/message_from_client/base_message.py
# ======================================================================
# Function: Message base class
# Author: Elvis Jia
# Date: 2016.6.27
# ======================================================================


class BaseMessage(object):

    def __init__(self, message_type="", target_type="", sequence_id=0):
        self.message_type = message_type
        self.target_type = target_type
        self.sequence_id = sequence_id
        self.message = ""
        self.success = False

File: GirlTest/Assets/Script/Message/BaseMessage.cs
using System;

[System.Serializable]
public class BaseMessage {
	// End mark
	public const string END_MARK = "/$/";
	// Message type
	public int message_type;
	// Message target type
	public int target_type;
	// Sequence id
	public int sequence_id;

	public BaseMessage(int message_type, int target_type, int sequence_id=0){
		this.message_type = message_type;
		this.target_type = target_type;
		this.sequence_id = sequence_id;
	}

	public BaseMessage(){
	}
}
